### DISCUSSION TITLE
What is the Laminar way of restricting access to a certain page?

### DISCUSSION THREAD

**Discussion by IgorWolkov** [raquo/Laminar#153] [Q&A] [answered]
Suppose I have a login and a content page, and I don't want to expose the internals of my js app to unauthorised users.
What is the Laminar way of restricting access to a content page? Shall I split the app on login app and content app and route it on a server side?

**Comment by raquo** [comment, marked as answer]
Yep, you need to split the Scala.js app. Basically you will have two entry
points instead of just one, and will create two JS bundles, one with the
login page, and another with the rest of the app. And the server needs to
serve the HTML file that points to one or the other JS bundle based on the
auth cookie or something like that.

**Comment by IgorWolkov** [comment]
@raquo, thank you very much.
